content/en/database_monitoring/guide/managed_authentication.md

```diff
@@ -61,14 +61,51 @@ AWS supports IAM authentication to RDS and Aurora databases. In order to configu
 }
 ```
 
+For example, if you wanted to use the `datadog` user, you would use the following resource ARN:
 
-3. Log in to your database instance as the root user, and grant the `rds_iam` role to the new user:
+```json
+{
+   "Version": "2012-10-17",
+   "Statement": [
+       {
+           "Effect": "Allow",
+           "Action": [
+               "rds-db:connect"
+           ],
+           "Resource": [
+               "arn:aws:rds-db:REGION:ACCOUNT:dbuser:RESOURCE_ID/datadog"
+           ]
+       }
+   ]
+}
+```
+
+AWS also supports wildcards for specifying the resource, for example if you wanted to allow the `datadog` user to authenticate across all instances for an account add the following:
+
+```json
+  "Resource": [
+    "arn:aws:rds-db:*:ACCOUNT:dbuser:cluster-*/datadog",
+    "arn:aws:rds-db:*:ACCOUNT:dbuser:db-*/datadog"
+  ],
+```
+
+3. Log in to your database instance as the root user, and grant the `rds_iam` role[20] to the new user:
 
 
 ```tsql
+CREATE USER <YOUR_IAM_ROLE>;
 GRANT rds_iam TO <YOUR_IAM_ROLE>;
 ```
 
+For example, for the `datadog` user you would run:
+
+```tsql
```

Comment:
why tsql not sql?

Reply:
it doesnt matter, it just for the rendering & it looks the same